app/src/androidTest/java/org/systers/mentorship/SignUpActivityTest.kt

```diff
@@ -0,0 +1,146 @@
+package org.systers.mentorship
+
+import android.support.annotation.IdRes
+import android.support.design.widget.Snackbar
+import android.support.design.widget.TextInputLayout
+import android.support.test.espresso.Espresso
+import android.support.test.espresso.Espresso.onView
+import android.support.test.espresso.ViewInteraction
+import android.support.test.espresso.action.ViewActions.*
+import android.support.test.espresso.assertion.ViewAssertions.matches
+import android.support.test.espresso.matcher.ViewMatchers
+import android.support.test.espresso.matcher.ViewMatchers.*
+import android.support.test.rule.ActivityTestRule
+import android.support.test.runner.AndroidJUnit4
+import android.view.View
+import android.widget.EditText
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.Matchers
+import org.hamcrest.Matchers.allOf
+import org.hamcrest.TypeSafeMatcher
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotNull
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.systers.mentorship.view.activities.SignUpActivity
+
+/**
+ * This class specifies the UI test for SignUpActivity
+ */
+@RunWith(AndroidJUnit4::class)
+class SignUpActivityTest {
+
+    private val EMPTY_USERNAME_ERROR : String = "Username cannot be empty"
+    private val EMPTY_PASSWORD_ERROR : String = "Password cannot be empty"
+    private val EMPTY_EMAIL_ERROR : String = "Email cannot be empty"
+    private val EMPTY_NAME_ERROR : String = "Name cannot be empty"
+
+    /**
+     * This basically setups the SignUpActivity before test
+      */
+    @get:Rule
+    var mActivityRule : ActivityTestRule<SignUpActivity> = ActivityTestRule(SignUpActivity::class.java)
+
+    /**
+     * This method is used to find the EditText within the TextInputLayout. Useful for typing into the TextInputLayout
+     */
+    fun findEditTextInTextInputLayout(@IdRes textInputLayoutId : Int) : ViewInteraction {
+
+        return Espresso.onView(Matchers.allOf(ViewMatchers.isDescendantOfA(ViewMatchers.withId(textInputLayoutId)), ViewMatchers.isAssignableFrom(EditText::class.java)))
+    }
+
+
+    companion object {
+
+        /**
+         * This method simply implements the null check, checks the types and casts
+         */
+        fun hasTextInputLayoutErrorText(expectedErrorText : String) : Matcher<View> {
```

Comment:
keep the 1st ":" next to argument name:

```
expectedErrorText: String
```

Instead of this:

```
expectedErrorText : String
```

You can check this [Kotlin coding style convention here](https://kotlinlang.org/docs/reference/coding-conventions.html#class-header-formatting). See if you need to repeat this for other similar examples